code/time_interval.py

```diff
@@ -4,22 +4,24 @@
 ID_PROVIDER = IdProvider()
 
 class Time:
-    def __init__(self, hour: int, minute: int) -> None:
+    
+    # add new attribute "seconds"
+    def __init__(self, hour: int, minute: int, second: int) -> None:
         self.hour = hour
         self.minute = minute
+        self.second = second  
```

Comment:
Hmm, ich hatte mir das etwas anders vorgestellt. Und zwar wird Zeit in der Informatik immer als eine Zahl angegeben. Dabei wird der Startpunkt der Zeit festgelegt und von dort wird einfach angefangen zu zählen. 00:00 Uhr wäre in unserem Beispiel die Zahl 0. 00:01 Uhr wäre die 60, 01:00 Uhr wäre die 3600 und 13:29 Uhr die 48540. Wenn wir das als ganze Zahl speichern, ist auch das Umrechnen deutlich leichter.

Reply:
Du müsstest hier also alle Variablen löschen und nur noch die Variable `total_seconds` behalten. Der Trick bei der Sache: Bei den Funktionen und Methoden musst du die Paramter gar nicht ändern. Man kann hier vortäuschen, mit einer richtigen Zeitinterpretation zu abeiten, obwohl im Hintergrund nur eine ganze Zahl, die "Totalsekunde", steht.